src/app/pages/category/category.component.ts

```diff
@@ -30,36 +33,36 @@ export class CategoryComponent implements OnInit, AfterViewInit {
 	ngOnInit(): void {
 		this.products$ = combineLatest([
 			this.form.get('textControl')?.valueChanges as Observable<string>,
-			this.form.get('brandsControl')?.valueChanges as Observable<{}>,
+			this.brandsChanges$,
```

Comment:
this.form.valueChanges.pipe(
			map(() => this.form.get('homeInfo')),
			// Control
			distinctUntilChanged(),
			switchMap(() => (this.form.get('homeInfo') as FormGroup).valueChanges),
		);